src/coord/Weather.cc

```diff
@@ -0,0 +1,72 @@
+// -*- lsst-c++ -*-
+
+/*
+ * LSST Data Management System
+ * Copyright 2008, 2009, 2010 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+/**
+ * @file
+ * @brief Provide functions to handle dates
+ * @ingroup afw
+ * @author Steve Bickerton
+ *
+ */
+#include <sstream>
+#include <cmath>
+
+#include "lsst/pex/exceptions.h"
+#include "boost/format.hpp"
+#include "boost/algorithm/string.hpp"
+#include "boost/tuple/tuple.hpp"
+#include "lsst/afw/coord/Weather.h"
+
+namespace lsst { namespace afw { namespace coord {
+
+Weather::Weather(double airTemperature, double airPressure, double humidity) :
+    _airTemperature(airTemperature),
+    _airPressure(airPressure),
+    _humidity(humidity)
+{
+    validate();
+}
+
+bool Weather::operator==(Weather const & other) const {
+    return (_airTemperature == other.getAirTemperature()
+            && _airPressure == other.getAirPressure()
+            && _humidity == other.getHumidity());
```

Comment:
I thought that break after operator was in the coding standard.  It was certainly supposed to be!

Consistency is important, and the trailing operator makes it clear that the line is continued.